### PR TITLE
Combined multiple starter warnings into a single warning

### LINKED ISSUE #3424: Combine multiple starter warnings into one warning message area
When using the starter, if multiple error/help messages are shown, they are presented as two different error areas that require the user to close each message separately.  Is there a way to list these error messages in a single box/area instead?

Currently:

Image

Proposal (something like this but cleaner):

Image

### PR DESCRIPTION
**Pull request by SteveSamJacob19**
## Tested using browser:
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ ] Chrome (Desktop)

## Did you test accessibility:
- [ ] IBM Equal Access Accessibilty Checker
- [ ] Jaws (only relevant for new UX flows)

### CODE REVIEW

**Review by natalie-bernhard — CHANGES_REQUESTED** [review]
This branch needs to be rebased onto the current prod branch.  If you don't know about rebasing a branch, start with this article: https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase.  Let me know if you have any questions.

File: src/main/content/_assets/js/builds.js [hunk 1]
```diff
@@ -1106,19 +1106,22 @@ function displayMessage(message,javaMsg) {
...
    
-    $('#starter_warnings').append("<div class='" + classNameIs +"'>");
+    $('#starter_warnings').append("<li class='" + classNameIs +"'>");
  
     $('.' + classNameIs)
-    .append(message)
-    .append(close_icon);  
+    .append(message);
+
+    $('#starter_warnings li:first-child').css("padding-top","20px");
```
Comment (steven1046) [inline src/main/content/_assets/js/builds.js]: Can you move these styles to a css file since they don't seem to change based on logic

**Review by steven1046 — CHANGES_REQUESTED** [review]

**Comment by steven1046** [comment]
@SteveSamJacob19 can you please upload a screenshot of what the design looks like so @natalie-bernhard and others that view the history can see?

**Comment by SteveSamJacob19** [comment]
Screenshot 2023-11-27 at 9 08 24 PM

**Comment by natalie-bernhard** [comment]
Hi, can you make the spacing around the X more equal? Also, could you add a border-radius to the p element in the warning with a value of 7px so it matches the rounded appearance of the text boxes above it?

**Comment by SteveSamJacob19** [comment]
@natalie-bernhard I have made the requested changes

**Comment by natalie-bernhard** [comment]
> @natalie-bernhard I have made the requested changes

Can you add a new screenshot of the changes?

**Comment by SteveSamJacob19** [comment]
Screenshot 2023-12-04 at 12 19 14 PM
@natalie-bernhard Is this fine?

**Comment by natalie-bernhard** [comment]
Screenshot 2023-12-04 at 9 31 27 AM
The corners of the box should be curved like this from adding the border radius CSS rule mentioned in my earlier comment.

**Comment by SteveSamJacob19** [comment]
Screenshot 2023-12-06 at 11 51 40 AM
@natalie-bernhard How is this?

**Comment by natalie-bernhard** [comment]
Hi sorry to be picky, but can you decrease the spacing between the two lines of text? They look a little far apart at the moment.

**Comment by SteveSamJacob19** [comment]
@natalie-bernhard is this fine
Screenshot 2023-12-07 at 1 42 40 PM

**Review by steven1046 — APPROVED** [review]

**Comment by natalie-bernhard** [comment]
Yes this looks great, thank you! Merging this now as I had to do some git clean-up for the branch which led to the reviews getting messed up.

**Pull request merged**